File: code/d1.go
package main

import "fmt"

func main() {
// START OMIT
    var m map[string]int
    m = make(map[string]int)

    m["age"] = 33
    m["weight"] = 77

    for k, v := range m {
        fmt.Printf("%s %d\n", k, v)
    }

    delete(m, "age")
// END OMIT
}

File: code/d1.py
#!/usr/bin/env python

# START OMIT
d = {'name': 'koti', 'age': 28}
d['weight'] = 71.6

# Get only keys
for k in d.keys():
    print k

# Get only values
for v in d.values():
    print v

# Get both keys and values
for k, v in d.items():
    print k, v

del d['age']
# END OMIT
